pkg/deploy/api/types.go

```diff
@@ -115,6 +115,9 @@ type RollingDeploymentStrategyParams struct {
 	// TimeoutSeconds is the time to wait for updates before giving up. If the
 	// value is nil, a default will be used.
 	TimeoutSeconds *int64
+	// UpdatePercent is the percentage of replicas to scale up or down each
+	// interval. If nil, one replica will be scaled up and down each interval.
```

Comment:
The upstream code is hard-coded to scale by 1 up/down. I didn't want to change the existing default behavior to be a percent.